examples/lbuild.xml

```diff
@@ -0,0 +1,7 @@
+<library>
+  <repositories>
+    <repository><path>../repo.lb</path></repository>
+  </repositories>
+  <options></options>
+  <modules></modules>
```

Comment:
One could also put `<module>modm:build:scons</module>` in here, since it's used in every example, but I would add this in #73 instead.

I need to add a comment to this file, describing what it does, otherwise people will copy the examples over and it won't work anymore.